Azure/sonic-buildimage
# Clear VXLAN tunnel table from State DB on startup

#### Why I did it
When reloading config after crashes, VTEP interfaces are sometimes not created since the tunnel still exists in the STATE_DB.

See also https://github.com/Azure/sonic-buildimage/issues/9694#issuecomment-1022363135.
#### How I did it
Adding `VXLAN_TUNNEL_TABLE` to the list of tables to be cleaned in `swss.sh` fixes the problem.
#### How to verify it
* Setup EVPN-VXLAN fabric
* Crash orchagent
* Reload the config
* Observe missing VTEP interfaces
* Perform cold reboot
* Observe that VTEP interface gets created again
* Repeat all this with fix applied, observe that VTEP interfaces always get created now
#### Which release branch to backport (provide reason below if selected)

Reproduced in 202111 and 202106, didn't check the older releases but I'd expect them to have the same issue.

#### Description for the changelog
Clear VXLAN tunnel table from State DB on startup to prevent stale entries from blocking tunnel creation.

#### Link to config_db schema for YANG module changes
#### A picture of a cute animal (not mandatory but encouraged)
![Image from iOS](https://user-images.githubusercontent.com/10601319/168185914-34efd667-57f1-4019-b857-7ce62d5edfc4.jpg)
